pkg/services/featuremgmt/features.go

```diff
@@ -0,0 +1,95 @@
+package featuremgmt
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// FeatureToggleState indicates the quality level
+type FeatureToggleState int
+
+const (
+	// FeatureStateUnknown indicates that no state is specified
+	FeatureStateUnknown FeatureToggleState = iota
+
+	// FeatureStateAlpha the feature is in active development and may change at any time
+	FeatureStateAlpha
+
+	// FeatureStateBeta the feature is still in development, but settings will have migrations
+	FeatureStateBeta
+
+	// FeatureStateStable this is a stable feature
```

Comment:
What FeatureStateStable means btw? Does stable state means that this should be enabled by default or moved to its own config section from feature toggles list? Or it can stay here forever? It seems to me all feature toggles should be removed eventually.

Reply:
TBD -- but I think we can/should potentially use feature flags for things that are in development *and* releasable.  If this is all statically defined on startup it does not make much difference, but as we add support for the flag to be set for some users and not others, that seems like something we want to be applied more generally.

I am also nervous that we have many parallel growing lists of features that can be enabled, and hope this more robust structure can give us a powerful single place to improve support